.github/workflows/ci-tests.yml

```diff
@@ -176,7 +218,12 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: '3.8'
+        python-version: ${{ env.PYTHON_VERSION }}
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: ${{ env.pythonLocation }}
+        key: ${{ env.PYTHON_VERSION }}-${{ runner.os }}-build-${{ matrix.dbt-version }}-${{ hashFiles('tox.ini') }}
```

Comment:
Include `${{ hashFiles('setup.py') }}` and `${{ hashFiles('requirements_dev.txt') }}` in hash key